test/spec/ortbConverter/bidfloor_spec.js

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable no-trailing-spaces */
```

Comment:
this should be part of the PBS test suite, if we decide to do this only for PBS.